indico/modules/rb_new/client/js/modules/bookRoom/BookingSuggestion.jsx

```diff
@@ -125,6 +126,14 @@ export default class BookingSuggestion extends React.PureComponent {
                                       tooltip={PluralTranslate.string("We'll skip one occurrence to avoid any conflicts",
                                                                       "We'll skip {skip} occurrences to avoid any conflicts", skip, {skip})} />
                 )}
+                {shorten && (
+                    <SuggestionOption onClick={onClick}
+                                      overrides={{shorten}}
+                                      icon="calendar times"
+                                      text={PluralTranslate.string('Make booking non-recurring', 'Shorten by {shorten} days', shorten, {shorten})}
```

Comment:
Are you sure this is correct? If you shorten BY 1 occurrence it doesn't make it non-recurring. Also, you're shortening by days - unless it's a daily booking those counts are different.

Reply:
You are right. This is not what should be applied here at all. :eyes: It should be something like "Shorten by one day"